Add 'piecing-it-together' exercise (#2554)
exercism/problem-specifications: exercises/jigsaw-puzzle/introduction.md

# Introduction

> Your best friend has started collecting jigsaw puzzles and wants to build a detailed catalogue for their collection — recording information like the number of pieces, rows, columns, border pieces, inside pieces, aspect ratio, and the puzzle's format.

Reviewer: catalogue (UK) -> catalog (US)
```suggestion
Your best friend has started collecting jigsaw puzzles and wants to build a detailed catalog of their collection — recording information such as the total number of pieces, the number of rows and columns, the number of border and inside pieces, aspect ratio, and puzzle format.
```

> After solving one of the puzzles to figure out these details, you're now helping fill in the gaps for others in the collection, even when some data is missing.

Reviewer 2: I'm not great at this, but maybe this could be "spiced up" a little? To make it feel a little more whimsical.

Author: I tried my best.

# Introduction

> Your best friend has started collecting jigsaw puzzles and wants to build a detailed catalogue for the collection — recording information like the number of pieces, rows, columns, border pieces, inside pieces, aspect ratio, and the puzzle's format.

Reviewer 2: ```suggestion
Your best friend has started collecting jigsaw puzzles and wants to build a detailed catalogue for their collection — recording information like the number of pieces, rows, columns, border pieces, inside pieces, aspect ratio, and the puzzle's format.
```
